README.md

```diff
@@ -28,7 +28,7 @@ git checkout -t remotes/origin/release/0.25
 ```
 
 
-If you prefer to download an archive, you need to also download and place the submodule:
+If you prefer to download an archive, you also need to download and place the submodule:
```

Comment:
Old and new version both seem to be correct.

Reply:
Neither is incorrect but the latter reads more smoothly without the split infinitive.